dev-utils/tav-versions-fastify.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
```

Comment:
We have already a couple of files with the same logic. What do you think about a more generic script in a follow up issue?

Reply:
I'm fine either way. I am kind of hoping to spend some time on the side improving `tav` to support functionality like this natively. That's vapourware right now, however.